js/models/messages.js

```diff
@@ -1980,6 +1980,10 @@
           }
 
           let autoAccept = false;
+          const requestConversation = await ConversationController.getOrCreateAndWait(
```

Comment:
What is the difference between requestConversation and conversation? Also, there are a couple of ConversationController.getOrCreateAndWait(source, 'private').then() above, which point that we might as well define requestConversation earlier and use it in those places.

Reply:
Yeah not a good name, the reason I need to do this is that the conversation already defined by `const conversation = this.getConversation();` is retrieved using the conversationId which will always be the primary device conversation
The "requestConversation" is meant to be the conversation that belongs to the specific device that sent this message (made this request I guess I was thinking?) which may or may not be the primary device
Didn't see there were already getting the conversation from the source above (Mik added it, I'll assume there is a reason for doing that over the other conversation)
I'll change the name to `sendingDeviceConversation` unless there are any better ideas

Reply:
Ok, thanks for the explanation.
Yeah `sendingDeviceConversation` sounds good!